src/FeatureFlags/OptionBasedFormEditor/Settings/General.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Give\FeatureFlags\OptionBasedFormEditor\Settings;
+
+/**
+ * @unreleased
+ */
+class General extends AbstractOptionBasedFormEditorSettings
+{
+    /**
+     * @unreleased
+     */
+    public function getDisabledOptionIds(): array
+    {
+        return [
+            //General Section
+            'success_page',
+            'failure_page',
+            // Currency Section
+            'auto_format_currency',
+            'currency_position',
+            'thousands_separator',
+            'decimal_separator',
+            'number_decimals',
+            'currency_preview',
+        ];
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getDisabledSectionIds(): array
+    {
+        return [
+            'access-control',
```

Comment:
So I think we actually need to double check this section.  I know I originally thought these were legacy but now I'm pretty sure some of the settings here are actually used in the donor dashboard like the "Email access" setting.

Reply:
Resolved in af1cd458b17269d5f81a62c0c062e0059823cf1f and 7c3467d3a59c972e76f51f669671ccc8b48cf18c